.env.example

```diff
@@ -15,3 +15,8 @@ REACT_APP_SNET_SIGNER_ADDRESS=
 REACT_APP_MPE_CONTRACT_ADDRESS=
 REACT_APP_EXECUTOR_WALLET_ADDRESS=
 REACT_APP_SNET_SUPPORT_MAIL=
+
+## Sandbox variables
```

Comment:
We have these in a separate `.env` [file](https://github.com/singnet/snet-dapp/blob/master/.env.sandbox). And the steps to use sandbox mode is documented [here](https://github.com/singnet/snet-dapp/wiki/Sandbox-mode).

Reply:
[done](https://github.com/singnet/snet-dapp/pull/498/commits/6e46574b38a3144c1c944c7fe44bc6cf660f7f48)